core/views.py

```diff
@@ -74,6 +85,22 @@ def get_context_data(self, *args, **kwargs):
         context[self.tab_name] = True
         return context
 
+
+class AccountChangeView(AccountMixin, AccountTabsMixin, PermissionRequiredMixin, UpdateView):
+    template_name = 'core/user_accounts/account_edit.html'
+    form_class = AccountForm
+    success_url = reverse_lazy('core:user_accounts/account')
+    permission_required = ('core.can_view_account_information_self', 'core.can_change_account_information_self')
```

Comment:
These permissions don't actually exist. I suspect that you took the membership change view as a reference point for your changes, which do require specific permissions which do exist. Note how we create those in the model `membership_file.models.Member` using that model's [Meta option for `permissions`](https://docs.djangoproject.com/en/2.2/ref/models/options/#permissions)

Either way, I don't think we would want to limit account editing through permissions, so I'd suggest to remove `PermissionRequiredMixin` and its corresponding required permissions:
```suggestion

```